configurations/demo-config-1/locations/location-2/location-2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "tag": "location-2",
+  "objects": [
+    { "tag": "object-1", "position": { "row": 1, "col": 8 }, "type": "collectible", "assetPath": "assets/diamond.png" },
+    { "tag": "object-2", "position": { "row": 6, "col": 3 }, "type": "collectible", "assetPath": "assets/key.png" },
+    { "tag": "object-3", "position": { "row": 1, "col": 1 }, "type": "dialog", "assetPath": "assets/zombie.png" },
+    { "tag": "object-4", "position": { "row": 3, "col": 2 }, "type": "dialog", "assetPath": "assets/zombie.png" },
+    { "tag": "object-5", "position": { "row": 8, "col": 8 }, "type": "dialog", "assetPath": "assets/chest.png" }
+  ],
```

Comment:
some objects don't fit on the map. map size is currently dependent on the asset dimensions, it should probably be specified in config instead, but that's a problem for another task ;)